manual/RELEASE_HOWTO.md

```diff
@@ -48,6 +48,10 @@ Before merging PR:
     - Save draft
 - [ ] Change DOI in [`CITATION.cff`][citation] to new DOI
 - [ ] Change date-released in [`CITATION.cff`][citation]
+- [ ] Check `soname` in `makefile`:
+    - [ ] For patch and minor versions, check `abidiff` to see if `soname` needs
+          bumping
+    - [ ] For major versions, bump `soname`
```

Comment:
I guess we will always need to do it, but other libraries do have major releases with stable ABI.

I think 'Check `abidiff` to see if `soname` needs bumping' would do as well ...

Reply:
Yes, that's more concise, thanks!